cmake/FindStreamingDataTypes.cmake

```diff
@@ -1,53 +0,0 @@
-# Locates the streaming-data-types repository and sets up a target
-# called 'flatbuffers_generate' which generates the flatbuffer headers.
-# Make your target depend on 'flatbuffers_generate'.
-# When you call this using find_package, you can specify a minimum version
-# for the streaming data types:
-# find_package(streaming_data_types COMPONENTS <minimum-git-commit>)
-# you must then make your target depend on 'check_streaming_data_types'
-# so that it can abort compilation if streaming data types does not suffice.
-# Usage of this package requires of course the git suite to be available.
-# Contact: Dominik Werder
-
-find_path(path_include_streaming_data_types NAMES schemas/f141_epics_nt.fbs HINTS
```

Comment:
When was it decided that we do not support non-Conan builds any longer?
Removing this CMake module prevents a non-Conan build, or not?